### PR TITLE
Updates global lists to the global var subsystem.

### PR DESCRIPTION
**Pull request by DesolateG**
**What does this PR do?**
All global lists in the lists folder except for reagents.dm(which is being addressed in another PR) have been updated to use the global var defines, along with the strings that reference them. This should make it easier to maintain and ALOT easier to search for them, especially on the many generic list names.

**Changelog:**

:cl: DesolateG
tweak: Refactors global var lists to use the global subsystem.
/:cl:

### CODE REVIEW

**Comment by DesolateG** [comment]
Resolved the conflicts @variableundefined

**Comment by variableundefined** [comment]
Good heaven..

File: code/modules/admin/buildmode.dm [hunk 1]
```diff
@@ -0,0 +1,656 @@
+#define BASIC_BUILDMODE 1
+#define ADV_BUILDMODE 2
```
Comment (variableundefined) [inline code/modules/admin/buildmode.dm]: Hold up a moment, why is this included in the diff?

File: code/modules/mob/living/carbon/alien/larva/death.dm [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/mob/living/carbon/alien/larva/death(gibbed)
+	if(stat == DEAD)	return
```
Comment (variableundefined) [inline code/modules/mob/living/carbon/alien/larva/death.dm]: And this file too

Comment (DesolateG) [reply code/modules/mob/living/carbon/alien/larva/death.dm]: I can only assume these were files that were removed  with the buildmode and life/death refactoring.
@Crazylemon64  could confirm.

File: code/game/response_team.dm [hunk 3]
```diff
@@ -0,0 +1,777 @@
+// ERTs
```
Comment (DesolateG) [inline code/game/response_team.dm]: uh huh... i see...

**Comment by variableundefined** [comment]
This is the moment where after test merge you noticed no issue and then afterward you merge it in and everyone report 10,000 things and complain you merged it in.
